src/components/Card.tsx

```diff
@@ -16,6 +16,12 @@ function CharacterCard({ costume }: { costume: CostumeInfo }): JSX.Element {
       <a>
         <div className={"card " + "rarity" + costume.stars}>
           <div className="card-art">
+            <div className="element">
+              <img src="\icons\elements\dark.png"></img>
```

Comment:
Is there a reason you are using \ instead of / ?

Reply:
nope, good catch. surprised that even worked

Reply:
Yeah i'm surprised too! I believe it's Microsoft's way to handle paths so maybe it is handled by Node.js by default